src/config.ts

```diff
@@ -204,6 +207,31 @@ CQIDAQAB
     delete this.authCodeRequestParamsStore[authCode];
   }
 
+  public getAccessTokenFromStore(
+    clientIdSub: `${string}.${string}`
+  ): string[] | undefined {
+    return this.accessTokenStore[clientIdSub];
+  }
+
+  public addToAccessTokenStore(
+    clientIdSub: `${string}.${string}`,
+    accessToken: string
+  ): void {
+    this.accessTokenStore[clientIdSub] = [
+      ...(this.accessTokenStore[clientIdSub] ?? []),
+      accessToken,
+    ];
+  }
+
+  public deleteFromAccessTokenStore(
```

Comment:
I can't see an equivalent of this in authentication-api (specifically in AccessTokenService, which I believe these additions are replicating), and looks like you haven't used it anywhere here - what's the reason for including this method?

Reply:
Yeah added this, realised it we don't actually delete access tokens, and forgot to remove this. Have removed it!